model/src/main/java/org/cbioportal/genome_nexus/model/EnsemblCanonical.java

```diff
@@ -87,4 +89,14 @@ public String getEnsemblCanonicalGeneId() {
     public String getEnsemblCanonicalTranscriptId() {
         return this.ensemblCanonicalTranscriptId;
     }
+
+    public String getEntrezGeneId() {
+        // remove .0 for backwards compatibility with databases where
+        // entrez_gene_id field is a float
```

Comment:
python pandas library handles integer columns with missing values as floats, therefore the entrez gene id columns were stored as floats unf. Not that big of a deal, logged issue here: https://github.com/genome-nexus/genome-nexus-importer/issues/11